session2/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Consider adding a `lang` attribute to the html start tag to declare the language of this document.

This was a warning I got when I tried to validate the content of this HTML here: https://validator.w3.org/nu/#textarea

Feel free to use this tool in the future to validate your HTML and fix any warnings / errors that show up there.